go.mod needs to be updated to 1.14 or later

The go.mod references go version 1.13
https://github.com/elnormous/contenttype/blob/dcd30b18c159a48185c2fa84cf30d0194fbfc67c/go.mod#L3

and the code uses `net/http.Header.Values`, which was added in go 1.14
https://github.com/elnormous/contenttype/blob/dcd30b18c159a48185c2fa84cf30d0194fbfc67c/contenttype.go#L318-L323

Attempting to use the code or test it will result in:
```sh
$ go version
go version go1.13 linux/amd64

$ go test ./...
# github.com/elnormous/contenttype
./contenttype.go:355:38: request.Header.Values undefined (type http.Header has no field or method Values)
./contenttype.go:404:33: request.Header.Values undefined (type http.Header has no field or method Values)
FAIL    github.com/elnormous/contenttype [build failed]
FAIL
```

The minimal upgrade needed would be to increase it to go1.14.

--- a/go.mod
+++ b/go.mod
@@ -1,3 +1,3 @@
 module github.com/elnormous/contenttype
 
-go 1.13
+go 1.14

